engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/compiler/CompiledProcessWithDeps.scala

```diff
@@ -20,7 +21,7 @@ import scala.concurrent.duration.FiniteDuration
 
 class CompiledProcessWithDeps(compiledProcess: CompiledProcess,
                               val jobData: JobData,
-                              val exceptionHandler: FlinkEspExceptionHandler,
+                              exceptionHandler: FlinkEspExceptionHandler,
```

Comment:
I wonder, maybe we should make it () => FlinkEspExceptionHandler? Or add a comment that it has to be opened separately? (open in this clas does not handle it)

Reply:
Done, comment added. `exceptionHandler` so it can be only accessed via `prepareExceptionHandler`.